autogpt_platform/autogpt_libs/autogpt_libs/logging/config.py

```diff
@@ -97,26 +114,7 @@ def configure_logging(force_cloud_logging: bool = False) -> None:
             transport=SyncTransport,
         )
         cloud_handler.setLevel(config.level)
-        cloud_handler.setFormatter(StructuredLoggingFormatter())
```

Comment:
????????

Reply:
this was double wrapping messages. GCP logger has this included and our formatter was breaking the messages so they were unreadable in gcp